exdb/views.py

```diff
@@ -383,3 +385,36 @@ def get_context_data(self, *args, **kwargs):
         context = super(SearchExperienceResultsView, self).get_context_data(*args, **kwargs)
         context['search_query'] = self.request.GET.get('search', '')
         return context
+
+
+class SearchExperienceReport(CreateView):
+    access_level = 'basic'
+    keys = [
+        'name', 'status', 'author', 'planners', 'recognition', 'start_datetime',
+        'end_datetime', 'type', 'subtypes', 'description', 'goals', 'keywords',
+        'audience', 'guest', 'guest_office', 'attendance', 'created_datetime',
+        'next_approver', 'funds', 'conclusion',
+    ]
+
+    def get(self, *args, **kwargs):
+        if not self.request.GET.get('experiences'):
+            raise Http404
+        pks = json.loads(self.request.GET.get('experiences'))
+        if not pks:
+            raise Http404
+        experiences = Experience.objects.filter(pk__in=pks).prefetch_related(
```

Comment:
We should still exclude the same drafts that we exclude from the regular search page.